src/main/io/serial.c

```diff
@@ -511,12 +512,19 @@ static void nopConsumer(uint8_t data)
     UNUSED(data);
 }
 
+static void cbCtrlLine(void *context, uint16_t ctrl)
```

Comment:
This can be moved back to cli ... this is not strictly passthrough related (and baudrate callback is set there too)

Reply:
OK. Thanks for the feedback.